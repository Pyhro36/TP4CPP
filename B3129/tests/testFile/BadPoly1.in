PC convex0 5 6 3 6 3 -4 5 -4
PC nonConvex0 1 1 0 0 1 0 0 1
PC convex1 1 0 1 1 0 1 0 0
PC nonConvex1 1 3 8 11 8 5 2 11 10 7 10 4 7 2
PC nonConvex2 13 1 2 1 2 7 15 7 9 3
PC noPoly 1 10
PC noPoly -7 -8 5 8
PC convex2 17 4 8 0 2 0 0 7 2 10 11 11
PC convex3 -8 -6 -2 -10 1 -4 -5 -1
PC nonConvex3 -12 -5 -8 -3 -3 -9 -7 -11 -14 -7 -7 -1 -2 -6 -8 -9
PC nonConvex4 0 -1 6 7 -7 7 -7 -3 4 -3 4 -1
LIST
EXIT