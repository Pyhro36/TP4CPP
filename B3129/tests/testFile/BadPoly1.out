Shape handler 1.0 :
OK
ERR
#polygon defined is not convex
OK
ERR
#polygon defined is not convex
ERR
#polygon defined is not convex
ERR
#polygon defined contains less than 3 corners
ERR
#polygon defined contains less than 3 corners
OK
OK
ERR
#polygon defined is not convex
ERR
#polygon defined is not convex
PC convex0 5 6 3 6 3 -4 5 -4
PC convex1 1 0 1 1 0 1 0 0
PC convex2 17 4 8 0 2 0 0 7 2 10 11 11
PC convex3 -8 -6 -2 -10 1 -4 -5 -1

